src/e2e-test/java/io/cdap/plugin/utils/ElementHelper.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.utils;
+
+import io.cdap.e2e.utils.SeleniumDriver;
+import org.junit.Assert;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Element helper.
+ */
+public class ElementHelper {
+  private static final Logger logger = LoggerFactory.getLogger(PluginUtils.class);
+  private static final WebDriverWait webDriverWait = SeleniumDriver.waitDriver;
+  private static final JavascriptExecutor js = (JavascriptExecutor) SeleniumDriver.getDriver();
+  private static final Actions actions = new Actions(SeleniumDriver.getDriver());
+
+  // Element Actions
```

Comment:
These methods look generally useful (not specific to salesforce plugin). Can we add them in the framework if they're not already there?

Reply:
Updated. 

PR to update cdap-e2e-tests repo: https://github.com/cdapio/cdap-e2e-tests/pull/67 (It is in internal review right now.)

Reply:
do we need to wait for https://github.com/cdapio/cdap-e2e-tests/pull/67 to be merged before merging this PR?

Reply:
Yes, since the cdap-e2e-tests PR contains helper classes used in the current PR.

Reply:
New cdap-e2e-tests PR: https://github.com/cdapio/cdap-e2e-tests/pull/69